src/test/java/com/synopsys/integration/alert/web/controller/SystemControllerTestIT.java

```diff
@@ -94,61 +92,48 @@ public void testPostInitialSystemSetup() throws Exception {
     }
 
     @Test
     public void testGetSystemMessagesGetAfter() throws Exception {
-        ResponseFactory responseFactory = new ResponseFactory();
-        SystemController handler = new SystemController(systemActions, contentConverter, responseFactory);
-        ResponseEntity<String> responseEntity = handler.getSystemMessages("2018-11-13T00:00:00.000Z", null);
+        SystemController handler = new SystemController(systemActions);
+        handler.getSystemMessages("2018-11-13T00:00:00.000Z", null);
         Mockito.verify(systemActions).getSystemMessagesAfter(Mockito.anyString());
-        Mockito.verify(contentConverter).getJsonString(Mockito.any());
-        assertEquals(HttpStatus.OK, responseEntity.getStatusCode());
     }
 
     @Test
     public void testGetSystemMessagesGetBefore() throws Exception {
-        ResponseFactory responseFactory = new ResponseFactory();
-        SystemController handler = new SystemController(systemActions, contentConverter, responseFactory);
-        ResponseEntity<String> responseEntity = handler.getSystemMessages(null, "2018-11-13T00:00:00.000Z");
+        SystemController handler = new SystemController(systemActions);
+        handler.getSystemMessages(null, "2018-11-13T00:00:00.000Z");
         Mockito.verify(systemActions).getSystemMessagesBefore(Mockito.anyString());
-        Mockito.verify(contentConverter).getJsonString(Mockito.any());
-        assertEquals(HttpStatus.OK, responseEntity.getStatusCode());
     }
 
     @Test
     public void testGetSystemMessagesGetBetween() throws Exception {
-        ResponseFactory responseFactory = new ResponseFactory();
-        SystemController handler = new SystemController(systemActions, contentConverter, responseFactory);
-        ResponseEntity<String> responseEntity = handler.getSystemMessages("2018-11-13T00:00:00.000Z", "2018-11-13T01:00:00.000Z");
+        SystemController handler = new SystemController(systemActions);
+        handler.getSystemMessages("2018-11-13T00:00:00.000Z", "2018-11-13T01:00:00.000Z");
         Mockito.verify(systemActions).getSystemMessagesBetween(Mockito.anyString(), Mockito.anyString());
-        Mockito.verify(contentConverter).getJsonString(Mockito.any());
-        assertEquals(HttpStatus.OK, responseEntity.getStatusCode());
     }
 
     @Test
     public void testGetSystemMessagesBadDateRange() throws Exception {
-        ResponseFactory responseFactory = new ResponseFactory();
-        SystemController handler = new SystemController(systemActions, contentConverter, responseFactory);
+        SystemController handler = new SystemController(systemActions);
         Mockito.when(systemActions.getSystemMessagesBetween(Mockito.anyString(), Mockito.anyString())).thenThrow(new ParseException("error parsing date ", 0));
-        ResponseEntity<String> responseEntity = handler.getSystemMessages("bad-start-time", "bad-end-time");
+        try {
+            handler.getSystemMessages("bad-start-time", "bad-end-time");
```

Comment:
you could throw a failed assertion after this line if you're expecting this exception to be thrown